### DISCUSSION TITLE
Telecamera (ezviz) su Google Nest Hub

### DISCUSSION THREAD

**Discussion by Chiara1989** [indomus/forum#329] [HUB: Home Assistant]
Ciao a tutti,
vi chiedo cortesemente una mano... sembra una cosa banale ma non mi funziona...
L'idea è un'automazione che quando suona il citofono mi fa partire lo stram della telecamera in giardino (una ezviz) sul nest hub.
Il problema non è il trigger ma l'action, cioè non riesco a utilizzare i servizio camera.play_stream

Questo è quello che ho impostato
`service: camera.play_stream`
    `data:`
      `media_player: media_player.nest_cucina_2`
    `target:`
      `entity_id: camera.tommi`

Penserei che il flusso della cam non sia compatibile con il nest hub ma se lo richiamo dal nest hub stesso (non passando da HA) tutto funziona.
Quale impostazone sbaglio?
Grazie mille per la pazienza!!
Chiara

**Comment by seafrank** [comment]
stesso problema per me, per vedere il video su google nest devo prima attivarlo con "Hey google" dopo di che vedo lo streaming della mia telecamera, altrimenti il video rimane spento, questo pero' non succedeva quando ho implementato l'automatismo 6 mesi fa, pero' non so dire da quale aggiornamento non funziona piu'.
